exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/scan/ScanTestUtils.java

```diff
@@ -52,6 +52,8 @@
   public static final String FILE_PATH_COL = "filepath";
   public static final String SUFFIX_COL = "suffix";
   public static final String PARTITION_COL = "dir";
+  public static final String LMT_COL = "lmt";
```

Comment:
This one would benefit from documentation. "limit", "Last Metadata Time"? "Let Me Try"?